### ISSUE TITLE
Improvement Request

### ISSUE THREAD

**Issue by mrrosenthal** [softwareloop/uploader-plus#99]
We are using uploader plus and its great.  However, we have many documents that come into alfresco from different sources, such as redmine, odoo, a cmissync plugin etc.   They don't get uploaded in the normal manner inside alfresco.  

The improvement suggestion is-

If the documents doesnt get uploaded from the 'uploader' inside alfresco, documents without updated content types go into an inbox,  and then the user/manager/admin can know how many documents are without updated content types, and can then be aware to fix this problem.  

Basically, a catchall for ALL documents in alfresco without the updated content types.  

Thanks

**Comment by magp3** [comment]
@mrrosenthal Why not just make these documents that come from external sources set the proper type and metadata? If that's not an option why not make an "inbox" folder and make your external sources put the documents in that folder.

**Comment by douglascrp** [comment]
@mrrosenthal I have to agree with @magp3 
Your requirement has nothing to do with this addon.
You can solve your problem in lots of different ways.

@softwareloop Can you please close this issue?
